src/components/SelectComponents/SuggestionsList/SuggestionsList.tsx

```diff
@@ -93,7 +93,7 @@ export function SuggestionsList<S>({
                 return (
                     // eslint-disable-next-line react/no-array-index-key
                     <ListItem key={i}>
-                        <div {...elem('loaderItem')}>
+                        <div {...elem('loaderItem')} role="group">
```

Comment:
why did you decide to use this role here?

Reply:
I've replaced by listitem, i use it in Autosuggest.spec.tsx